web_action_conditionable/static/src/js/views.js

```diff
@@ -0,0 +1,23 @@
+/*global openerp, _, $ */
+
+openerp.web_action_conditionable = function (instance) {
+  instance.web.View.include({
+    is_action_enabled: function(action) {
+      var attrs = this.fields_view.arch.attrs;
+      if (action in attrs) {
+	try {
+	  data = JSON.parse(attrs[action]);
```

Comment:
you don't know what happens in the future, and also not what other modules exist that might also to whatever with this function. By calling super and only acting on failures, you give other code the chance to do whatever that code is good for